### PR TITLE
User view and create review of book

### PR DESCRIPTION
**Pull request by TaoLuyen**

### CODE REVIEW

File: app/views/reviews/_review.html.erb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    <p><%= t("reviews.content")%> <%= review.content %></p>
+    <p class="text-left display_inline"><em><%= t "reviews.created_by" %>
+      <%= link_to review.user.name, review.user %></em>
+    </p>
+    <p>
+      <% if can? :update, review %>
+        <%= link_to t("reviews.edit"), "#", class: "btn btn-primary" %>
+        <%= link_to t("reviews.delete"), "#", method: :delete,
+          data: {confirm: t("reviews.confirm")}, class: "btn btn-danger" %>
+        <% end %>
```
Comment (hangxuka) [inline app/views/reviews/_review.html.erb]: ident

File: app/views/reviews/_review.html.erb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  <% @reviews.each do |review| %>
+  <div id="sub_review">
+    <h4 class="display_inline"><strong><%= review.user.name %></strong></h4>
+
+    <div class="show-rating-review">
+      <%= t("reviews.rate")%>
+      <%= review.rating %>
+    </div>
+    <p><%= t("reviews.content")%> <%= review.content %></p>
+    <p class="text-left display_inline"><em><%= t "reviews.created_by" %>
```
Comment (hangxuka) [inline app/views/reviews/_review.html.erb]: cho <em> xuong dong se nhin hay hơn

**Comment by hangxuka** [comment]
ok

File: app/models/review.rb [hunk 3]
```diff
@@ -3,4 +3,16 @@ class Review < ActiveRecord::Base
...
   has_many :comments
+
+  validates :content, presence: true
+  validate :check_rating
+
+  scope :order_reviews, ->{order created_at: :DESC}
+
+  private
+  def check_rating
+    if((rating < 1) || (rating > 10))
```
Comment (dieunb) [inline app/models/review.rb]: `if rating < 1 || rating > 10`

File: app/models/review.rb [hunk 4]
```diff
@@ -3,4 +3,16 @@ class Review < ActiveRecord::Base
...
+      errors.add(:picture, I18n.t("reviews.validate_rating"))
```
Comment (dieunb) [inline app/models/review.rb]: `errors.add :picture, I18n.t("reviews.validate_rating")`

File: app/models/review.rb [hunk 5]
```diff
@@ -3,4 +3,16 @@ class Review < ActiveRecord::Base
...
+
+  validates :content, presence: true
+  validate :check_rating
+
+  scope :order_reviews, ->{order created_at: :DESC}
+
+  private
+  def check_rating
+    if rating < 1 || rating > 10
+      errors.add(:rating, I18n.t("reviews.validate_rating"))
```
Comment (dieunb) [inline app/models/review.rb]: `errors.add :rating, I18n.t("reviews.validate_rating")`

**Pull request merged**